packages/ibm-products/src/components/PageHeader/PageHeader.tsx

```diff
@@ -1035,13 +1044,32 @@ export let PageHeader = React.forwardRef(
                 </Row>
               ) : null}
 
-              {subtitle ? (
+              {subtitle && (
                 <Row className={`${blockClass}__subtitle-row`}>
                   <Column className={`${blockClass}__subtitle`}>
-                    {subtitle}
+                    {isOverflowing ? (
+                      <DefinitionTooltip
+                        definition={subtitle}
+                        className={`${blockClass}__subtitle-tooltip`}
+                      >
+                        <span
+                          ref={overflowSubtitleRef}
+                          className={`${blockClass}__subtitle-text`}
+                        >
+                          {subtitle}
+                        </span>
```

Comment:
You could create a function to return this jsx and use it below as well, just to cut down on some repitition